frontend/src/HomePage.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { useEffect, useState } from "react";
+import { NavLink, Route, Routes } from "react-router-dom";
+import TravelerGroupOverview from "./components/TravelerGroup/TravelerGroupOverview";
+import { TravelerGroupModel } from "./components/TravelerGroup/TravelerGroupModel/TravelerGroupModel";
+import { TravelerModel } from "./components/Traveler/TravelerModel/TravelerModel";
+import axios from "axios";
+import AddTravelerGroupForm from "./components/TravelerGroup/AddTravelerGroupForm/AddTravelerGroupForm";
+import AddTravelerForm from "./components/Traveler/AddTravelerForm/AddTravelerForm";
+import TravelerOverview from "./components/Traveler/TravelerOverview";
+import "./Homepage.css";
+
+export default function HomePage() {
+  const [travelerGroupList, setTravelerGroupList] = useState<
+    TravelerGroupModel[]
+  >([]);
+  const [travelers, setTravelers] = useState<TravelerModel[]>([]);
+
+  useEffect(() => {
+    fetchAllTravelers();
+    fetchAllTravelerGroups();
+  }, []);
+
+  const fetchAllTravelerGroups = () => {
+    axios
+      .get("/api/traveler-groups")
+      .then((response) => {
+        return response.data;
+      })
+      .then(setTravelerGroupList);
+  };
+  const fetchAllTravelers = () => {
+    axios
+      .get("/api/travelers")
+      .then((response) => response.data)
+      .then(setTravelers);
+  };
```

Comment:
Consider having catch-statements in your axios call like
`catch((error) => console.error("Error while getting xyz:" + error))`